include/treelite/annotator.h

```diff
@@ -27,14 +29,14 @@ class BranchAnnotator {
   void Annotate(const Model& model, const DMatrix* dmat, int nthread, int verbose);
   /*!
    * \brief load branch annotation from a JSON file
-   * \param fi input stream
+   * \param fp input stream
    */
-  void Load(dmlc::Stream* fi);
+  void Load(FILE* fp);
```

Comment:
Why not c++ stream?

Reply:
Good point. Let me use `std::istream&` here, to be consistent with the changes made in `frontend/xgboost.cc` and `frontend/lightgbm.cc`.